pkg/schedulingtypes/resources.go

```diff
@@ -43,6 +45,16 @@ var PodResource = &metav1.APIResource{
 	Namespaced: true,
 }
 
+var JobResource = &metav1.APIResource{
+	Name:       GetPluralName(Job),
+	Group:      batchv1.SchemeGroupVersion.Group,
+	Version:    batchv1.SchemeGroupVersion.Version,
+	Kind:       Job,
+	Namespaced: true,
+}
+
+// The resource name key is the corresponding federated resource name
+// for the given k8s resource definition.
 var ReplicaSechedulingResources = map[string]metav1.APIResource{
```

Comment:
There's a typo while assigning a variable. It should be **ReplicaSchedulingResources**.